src/renderer/src/lib.rs

```diff
@@ -12,15 +12,14 @@ extern crate mopa;
 
 extern crate glutin;
 extern crate cgmath;
-
-extern crate amethyst_ecs;
+extern crate specs;
 
 /// Contains the included Render Targets
 pub mod target;
 /// Contains the included Passes
 pub mod pass;
 
-use amethyst_ecs::{Component, VecStorage};
+use self::specs::{Component, VecStorage};
```

Comment:
I've messed around with implementing external traits on external types, and there doesn't seem to be any hacks or workarounds. Thanks for directing me to rust-lang/rfcs#493; I'll be following the issue very closely! In the meantime, I don't want to hold up the review process. We'll hopefully revisit this later in the future if and when Rust catches up. You've got my approval! 😄